words.yaml

```diff
@@ -268,6 +268,12 @@ djo:
   eng_long: ___ (donor) gives/donates gift/present ___ to recipient ___ (without payment/exchange)
   eng_short: give
   jbo_similar: dunda
+dju:
+  _family: "R"
+  _signature: ".."
+  eng_long: "___ knows facts about ___"
```

Comment:
If I "know your dog", i don't mean I know "facts about your dog". Facts can be expressed with djun. djun should be `___ knows ___`.

Reply:
Sounds good.